src/EduManagementLab.Api/Controllers/CoursesController.cs

```diff
@@ -181,15 +193,25 @@ public ActionResult<List<Course.Membership>> GetCourseMemberships(Guid courseId)
         }
 
         [HttpGet]
+        [ProducesResponseType(typeof(List<MembershipDto>), StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
         [Route("{courseId}/Memberships/{userId}")]
-        public ActionResult<List<Course.Membership>> GetCourseMemberships(Guid courseId, Guid userId)
+        public ActionResult<List<MembershipDto>> GetCourseMemberships(Guid courseId, Guid userId)
```

Comment:
Bug merged in a previous pull request.
I dont think we want this endpoint. If we want to expose the memberships for the user we add this endpoint to the User controller.
{userId}/CourseMemberships

Reply:
Hope you saw my edit above.

Reply:
Changed the method, but it could use the addition of a core method to retrieve selected membership to have cleaner code.